tools/setup_stackdriver.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+    cat <<EOF
+
+Generate stackdriver.yaml file to enable stackdriver handler for istio.
+
+usage: ${0} [OPTIONS]
+
+The following flags are required.
+
+       --project_id       Project Id of the GCP Project.
+       --sink_id          Id of the sink. Ex: temp-gcs
+       --sink_destination Destination Id of the sink. Ex: storage.googleapis.com/temp_gcs
+       --log_filter      Filter on how to filter logs for the sink. Ex: severity >= DEFAULT
+EOF
+    exit 1
+}
+
+while [[ $# -gt 0 ]]; do
+    case ${1} in
+        --project_id)
+            project_id="$2"
+            shift
+            ;;
+        --sink_id)
+            sink_id="$2"
+            shift
+            ;;
+        --sink_destination)
+            sink_destination="$2"
+            shift
+            ;;
+        --log_filter)
+            log_filter="$2"
+            shift
+            ;;
+        *)
+            usage
+            ;;
+    esac
+    shift
+done
+
+# We check for non-empty for project_id as it's required to setup stackdriver for the project.
+if [[ $project_id = '' ]] ; then
+  echo "Project Id can't be empty. Please provide a valid project id."
+  exit 1
+fi
+
+
+{
+    cat <<EOF > stackdriver.yaml
```

Comment:
can this YAML file live elsewhere and the script just consume it? I'm worried about finding all the places to update later.